### PR TITLE
added css to dropdown an account field and edit it.  No functionality…

### PR DESCRIPTION
**Pull request by stevenyee99**
… on change or click since Im not sure exactly how were handling questions.  BTW React Hooks are awesome!!!

### CODE REVIEW

File: src/assets/icons/Arrow.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ import React from 'react';
 export default ({ fillColor, size }) => {
     return (
         <svg  fill={fillColor || 'black'} width={size || "24"} height={size || "24"} viewBox="0 0 24 24">
-            <path d="M5 3l3.057-3 11.943 12-11.943 12-3.057-3 9-9z"/>
```
Comment (stevenyee99) [inline src/assets/icons/Arrow.js]: Swapped old svg with new Material Design svg

File: src/components/my-account/SectionBody.js [hunk 2]
```diff
@@ -1,17 +1,53 @@
...
+import React, { useState } from 'react';
 import ArrowIcon from '../../assets/icons/Arrow';
+import DownArrowIcon from '../../assets/icons/DownArrow';
+
+import Button from '@material-ui/core/Button';
+// Temp for now.  I should get a question and render it
+import TextField from '@material-ui/core/TextField';
 
 export default ({ name, label }) => {
+    const [editOpen, setEditOpen] = useState(false);
```
Comment (stevenyee99) [inline src/components/my-account/SectionBody.js]: This is hw you use hooks.  Super easy way to do state without the cruft of a component.  #thisIsTheFuture
https://reactjs.org/docs/hooks-overview.html

File: src/components/my-account/SectionBody.js [hunk 3]
```diff
@@ -1,17 +1,53 @@
...
+            <div className="my-account-section">
+                <p className="my-account-section-name">{name}</p>
+                <div className="my-account-section-right">
+                    <p className="my-account-section-value">{label}</p>
+                    <a className="my-account-section-arrow" onClick={() => setEditOpen(!editOpen)}>
+                        {editOpen ? 
+                            <DownArrowIcon size='36' fillColor='lightGray' />
+                            : 
+                            <ArrowIcon size='36' fillColor='lightGray' />
+                        }
```
Comment (stevenyee99) [inline src/components/my-account/SectionBody.js]: THis is a ternary operator.  
Sugar for if (condition) { A } else { B }

File: src/components/my-account/SectionBody.js [hunk 4]
```diff
@@ -1,17 +1,53 @@
...
-                <p className="my-account-section-value">{label}</p>
-                <a className="my-account-section-arrow" onClick={() => console.log('arrow icon clicked')}>
-                    <ArrowIcon size='16' fillColor='lightGray' />
-                </a>
+        <div>
+            <div className="my-account-section">
+                <p className="my-account-section-name">{name}</p>
+                <div className="my-account-section-right">
+                    <p className="my-account-section-value">{label}</p>
+                    <a className="my-account-section-arrow" onClick={() => setEditOpen(!editOpen)}>
```
Comment (stevenyee99) [inline src/components/my-account/SectionBody.js]: Implementation of hooks in action!.
setEditOpen is basically this.setState( { editOpen: !this.state.editOpen })

File: src/components/my-account/SectionBody.js [hunk 5]
```diff
@@ -1,17 +1,53 @@
...
+            {editOpen && (
+                <div className="my-account-section-edit row">   
+                    <div className="col-xs-5">
+                        <TextField
+                            className="my-account-section-edit-question"
+                            label={name}
+                            value={'lol'}
+                            helperText={''}
+                            onChange={() => console.log('answer changed')}
+                            margin="normal"
```
Comment (stevenyee99) [inline src/components/my-account/SectionBody.js]: I'ma need to discuss how you want to sync and structure data for you account questions before I can hook this up

File: src/assets/icons/Arrow.js (hunk 1 above)
Comment (owlsandals) [reply src/assets/icons/Arrow.js]: Where do these come from?

File: src/components/my-account/my-account.less [hunk 6]
```diff
@@ -79,6 +79,15 @@
...
         }
+        &-edit {
+            margin-bottom: 1rem;
+            &-question {
+                margin-left: 2rem !important;
+            }
+            &-save {
+                margin: auto;
+            }
+        }
```
Comment (stevenyee99) [inline src/components/my-account/my-account.less]: I found a bug with npm run dev. . . it didnt get my css changes.  Will look into eventualy

File: src/assets/icons/Arrow.js (hunk 1 above)
Comment (stevenyee99) [reply src/assets/icons/Arrow.js]: https://material.io/tools/icons/?icon=chevron_right&style=baseline

**Pull request merged**